application/review/views.py

```diff
@@ -41,12 +47,12 @@ def review_page(review_token):
         abort(404)
 
 
-@review_blueprint.route("/new-review-url/<id>/<version>")
+@review_blueprint.route("/new-review-url/<id>")
 @login_required
-def get_new_review_url(id, version):
+def get_new_review_url(id):
```

Comment:
This feels like it has fairly broken access controls. We should add a `current_user.can_access(measure.slug)` check in there, at the least. Possibly also a `@user_can(CREATE_MEASURE)` decorator?

Reply:
Good point.